run.sh

```diff
@@ -1,8 +1,64 @@
 #!/bin/sh
 
+#plagiarized from wangxian/alpine-mysql
```

Comment:
This should be in it's own separate file for Docker as well because you're making a lot of assumptions about the user environment that the upstream script shouldn't make. For example that the user is even using MySQL since CyTube is targeting Knex.js now and the user could be using Postgres.